index.html

```diff
@@ -169,66 +199,97 @@ <h6>Back End Dev</h6>
+            <a href="https://medium.com/@zermarden"
+              ><img src="assets/m.svg" alt="Medium" width="20" height="20"
+            /></a>
+          </li>
         </ul>
         <button>Get my resume</button>
         <ul class="langs">
           <li class="lang">
-            <h5>
-              Languages
-            </h5>
-            <img class="disabled" src="assets/Disabled.svg" alt="disabled">
+            <h5>Languages</h5>
+            <img class="disabled" src="assets/Disabled.svg" alt="disabled" />
           </li>
-          <li>  
+          <li>
             <ul class="lang-list">
               <li>
-                <img src="assets/javascript.svg" alt="JavaScript">
-                <h6>
-                  JavaScript
-                </h6>
+                <img src="assets/javascript.svg" alt="JavaScript" />
+                <h6>JavaScript</h6>
               </li>
               <li>
-                <img src="assets/HTML-5.svg" alt="HTML">
-                <h6>
-                  HTML-5
-                </h6>
+                <img src="assets/HTML-5.svg" alt="HTML" />
+                <h6>HTML-5</h6>
               </li>
               <li>
-                <img src="assets/CSS-3.svg" alt="CSS">
-                <h6>
-                CSS-3
-                </h6>
+                <img src="assets/CSS-3.svg" alt="CSS" />
+                <h6>CSS-3</h6>
               </li>
             </ul>
           </li>
           <li class="lang">
-            <h5>
-              Frameworks
-            </h5>
-            <img class="chevron" src="assets/chevron.svg"  alt="chevron">
+            <h5>Frameworks</h5>
+            <img class="chevron" src="assets/chevron.svg" alt="chevron" />
           </li>
           <div class="solid"></div>
           <li class="lang">
             <h5>Skills</h5>
-              <img class="chevron" src="assets/chevron.svg" alt="chevron">
+            <img class="chevron" src="assets/chevron.svg" alt="chevron" />
           </li>
           <div class="solid"></div>
         </ul>
       </section>
+
+      <!--Contact Me Section-->
+
+      <section class="contact-form">
+        <div class="form-container">
+          <div class="container1">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a feature that
+              you need built or a project that needs coding. I’d love to help with it
+            </p>
+          </div>
+          <div class="container2">
+            <form action="https://formspree.io/f/xgedddaq" method="POST">
+              <input type="text" name="name" maxlength="30" placeholder="Your name" required />
+              <input type="email" name="email" placeholder="Email address" required />
+              <textarea name="message" maxlength="500" placeholder="Enter your message here" required>
+              </textarea>
+              <input class="button" type="submit" value="Get In Touch">
```

Comment:
- [ ] Kindly, make sure that Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. check the screenshot below:
![image](https://user-images.githubusercontent.com/71874425/186142037-41a9e629-218f-4ab5-9f54-cf90ae740945.png)

Reply:
Thank you very much for your friendly and professional review!